src/lib/customPropTypes.tsx

```diff
@@ -202,7 +206,7 @@ export const some = validators => (props, propName, componentName, ...rest) => {
   // fail only if all validators failed
   if (errors.length === validators.length) {
     const error = new Error('One of these validators must pass:')
-    error.message += `\n${_.map(errors, (err, i) => `[${i + 1}]: ${err.message}`).join('\n')}`
+    error.message += `\n${map(errors, (err, i) => `[${i + 1}]: ${err.message}`).join('\n')}`
```

Comment:
Let's also retain `lodash/fp` here.  In this case, we need to drop the `i` (index) since `lodash/fp`'s map iterator is a unary function:

```diff
-map(errors, (err, i) => `[${i + 1}]: ${err.message}
+_.map(errors, err => `- ${err.message}`)
```

We don't need the fancy number in brackets ;)

Reply:
done, thanks